subprojects/core-api/src/main/java/org/gradle/vcs/VersionRef.java

```diff
@@ -26,6 +26,13 @@
  */
 @Incubating
 public interface VersionRef {
+    /**
```

Comment:
I don't think this `String` should be part of our user-facing API. If it should be, then it should not be part of the `VersionRef` as that is a concept specific to version-control systems whereas `latest.integration` is a concern of the more general dependency resolution process. For example, it might make sense as a constant on some type related to Versions of Components, but not here.

Reply:
+1, since this is a public API, I think it's better for this to remain a private thing in the VcsDependencyResolver.

Reply:
Make sense, done.